components/PoweredBy/powered-by.css

```diff
@@ -0,0 +1,12 @@
+.ais-powered-by {
+  display: block;
```

Comment:
not sure we need `display:block`, it's the default right? It should work without it